modules/bot/models/User.php

```diff
@@ -121,10 +121,4 @@ public function save($runValidation = true, $attributeNames = null)
         $this->state = $this->getState()->toJson();
         return parent::save($runValidation, $attributeNames);
     }
-
-    public function getChats()
-    {
-        return $this->hasMany(Chat::className(), ['id' => 'chat_id'])
-                    ->viaTable('bot_chat_bot_user', ['user_id' => 'id']);
-    }
```

Comment:
Rollback this change and make this method work.
Tip: change 'bot_chat_bot_user' to 'bot_chat_member'